src/util/environment.ts

```diff
@@ -19,25 +19,51 @@ export const getDriverEnv = (): string => {
     runtime: "unknown",
   };
 
-  let isServiceWorker = false;
-  try {
-    isServiceWorker = global instanceof ServiceWorkerGlobalScope;
-  } catch (_) {
-    // okay if this fails
-  }
```

Comment:
are we losing some safety?

Reply:
No, we should be fine. I found recommendations for checking for Service Worker env that don't rely on trying something that can throw. This try/catch block was only around using `instanceof`. `typeof` doesn't throw an error if the variable isn't declared.